src/core/OmniElement.ts

```diff
@@ -93,17 +93,27 @@ export abstract class OmniElement extends LitElement {
                 --omni-border-radius: var(--omni-theme-border-radius, 4px);
                 --omni-border-width: var(--omni-theme-border-width, 2px);
 
+                /* global z-index pattern for elements */
+                --omni-z-index-item-container: 400;
+                --omni-z-index-dialog: 500;
+                --omni-z-index-dropdown: 600;
+                --omni-z-index-alert: 700;
+                --omni-z-index-tooltip: 800;
+
+                /* global z-index for component elements */
+                --omni-z-index-form-label: 410;
```

Comment:
These should expose variables as well. Are we using all of these? Are they really core theme variables or only used by a specific component?

Reply:
Removing the global variables as they are not used by components it should be discussed in future as more components are added. The --omni-z-index-form-label will be removed and the user should have the option to override the OmniFormElement css variable.